pkg/k8shandler/collection.go

```diff
@@ -149,6 +153,51 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateCollectionPriorityCla
 	return nil
 }
 
+func (clusterRequest *ClusterLoggingRequest) createCollectorMetricsRBAC() error {
+	clusterRole, err := clusterRequest.CreateClusterRole(
+		"log-collector-metrics",
```

Comment:
shouldn't this be created as part of the manifests that deploy the operator?
Giving the operator permission to create clusterroles doesn't seem like a good idea :confused:

Reply:
learned that OLM supports creation of resources from manifest files. modified accordingly

Reply:
Thanks! that's way better!